l10n_fr_state/migrations/9.0.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# © 2015-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+def migrate(cr, version):
+    if not version:
+        return
+
+    # key : source region XMLID - value : destination region XMLID
+    new_region_map = {
+        'res_country_state_rhonealpes': 'res_country_state_auvergne',
```

Comment:
I don't think that xmlid should be changed (at least not changed in a stable version) ; that's why the xmlid were left unchanged in v8 (this PR is just a forward-port of the changes in v8). xmlid is a technical name visible only to the developers ; if someone rely on the current xmlid, it will break his code, and we don't want that in a stable branch.